run.py

```diff
@@ -126,14 +138,16 @@ def __rdf_from_dataset(self, dataset):
         else:
             raise NameError(
                 'Impossible to create RDataFrame with different tree names')
-        chain = TChain(tree_name, tree_name)
+        chain = TChain()
```

Comment:
Is this way to setup a chain better or worse than the other? Do we know?

Reply:
I wrote it here https://github.com/KIT-CMS/ntuple_processor/issues/24

Reply:
Alright, thanks!